bot/cogs/games/games.py

```diff
@@ -39,3 +39,19 @@ async def ball8(self, ctx: Context, *, question: str) -> None:
         embed.add_field(name="Answer", value=answer)
         
         await ctx.send(embed=embed)
+    @command()
```

Comment:
you need to configure an alias for faster usage, instead of keeping just this long name
```suggestion
    @command(aliases=["cflip"])
```